src/components/ProposalForm.tsx

```diff
@@ -45,31 +85,47 @@ export default function ProposalForm() {
           </CardDescription>
         </CardHeader>
         <CardContent>
-          <form className="space-y-4">
+          <form className="space-y-4" id="proposalForm" onSubmit={onSubmit}>
             <div className="grid grid-cols-2 gap-4">
               <div className="space-y-2">
                 <Label htmlFor="name">Name</Label>
-                <Input id="name" placeholder="Your Name" required />
+                <Input
+                  id="name"
+                  placeholder="Your Name"
+                  type="text"
+                  name="authorName"
+                  required
+                  minLength={6}
+                />
               </div>
-              <div className="space-y-2">
+              <div id="email-field" className="space-y-2">
                 <Label htmlFor="email">Email</Label>
                 <Input
                   id="email"
+                  name="authorEmail"
                   placeholder="you@example.com"
                   required
                   type="email"
+                  pattern=".+@.+"
```

Comment:
SUGGESTION: As it turns out, this `<Input/>` component already does some pretty nice email validation for us because we set `type="email"`, so we can actually remove this line altogether
```suggestion
```